x-pack/plugins/infra/public/pages/metrics/hosts/components/tabs/metrics/metric_chart.tsx

```diff
@@ -130,7 +130,7 @@ export const MetricChart = ({ id, title, layers, overrides }: MetricChartProps)
         </EuiFlexGroup>
       ) : (
         <LensWrapper
-          id={`hostsViewsmetricsChart-${id}`}
+          id={`hostsViewMetricsChart-${id}`}
```

Comment:
Q: Is this affecting the tests? Or it's used only for telemetry 🤔

Reply:
I don't think this is being used, honestly. It's neither in telemetry [spreadsheet](https://docs.google.com/spreadsheets/d/1h_2-wjgupJugoChhFKnOcZqqJwlZ2H5Olpj89_bpHXY/edit#gid=0) nor referenced in the tests.